commands/init/deployer/compile-deployer.ts

```diff
@@ -7,6 +7,16 @@ const injectSC = (scFilePath: string): string => {
     return deployer.replace("##Wasm_file_path##", scFilePath);
 };
 
+const injectInit = (deployer: string): string => {
+    const initFile = "./assembly/init.ts";
+    if (fs.existsSync(initFile)) {
+        const init = fs.readFileSync("./assembly/init.ts", "utf-8");
+        return deployer.replace("//##init_function##", init).replace("//##init_call##", "init(contractAddr);");
```

Comment:
When are trying to avoid token/regexp replacer. We have seen in the past that it's can be buggy. Can you use a replacer instead?

Reply:
I resolve because it's outdated. We use a new way now.